prov/efa/src/rxr/rxr_pkt_type_rts.c

```diff
@@ -223,32 +223,12 @@ ssize_t rxr_pkt_init_rts(struct rxr_ep *ep,
 	assert(peer);
 	data = src;
 	rts_hdr = rxr_get_rts_hdr(pkt_entry->pkt);
-	if (rxr_env.enable_shm_transfer && peer->is_local) {
-		rts_hdr->flags |= RXR_SHM_HDR;
-		/* will be sent over shm provider */
-		if (tx_entry->total_len <= rxr_env.shm_max_medium_size) {
-			data_len = ofi_copy_from_iov(data, rxr_env.shm_max_medium_size,
-						     tx_entry->iov, tx_entry->iov_count, 0);
-			assert(data_len == tx_entry->total_len);
-			rts_hdr->flags |= RXR_SHM_HDR_DATA;
-			pkt_entry->pkt_size += data_len;
-		} else {
-			/* rendezvous protocol
-			 * place iov_count first, then local iov
-			 */
-			memcpy(data, &tx_entry->iov_count, sizeof(size_t));
-			data += sizeof(size_t);
-			pkt_entry->pkt_size += sizeof(size_t);
-			memcpy(data, tx_entry->iov, sizeof(struct iovec) * tx_entry->iov_count);
-			pkt_entry->pkt_size += sizeof(struct iovec) * tx_entry->iov_count;
-		}
-	} else {
-		/* will be sent over efa provider */
-		data_len = ofi_copy_from_iov(data, mtu - pkt_entry->pkt_size,
-					     tx_entry->iov, tx_entry->iov_count, 0);
-		assert(data_len == rxr_get_rts_data_size(ep, rts_hdr));
-		pkt_entry->pkt_size += data_len;
-	}
+
+	/* will be sent over efa provider */
+	data_len = ofi_copy_from_iov(data, mtu - pkt_entry->pkt_size,
+				     tx_entry->iov, tx_entry->iov_count, 0);
+	assert(data_len == rxr_get_rts_data_size(ep, rts_hdr));
+	pkt_entry->pkt_size += data_len;
```

Comment:
RTS is not used any more. Any change to this file is unnecessary.

Reply:
I only remove the shm related logic here, and didn't touch the efa part. If we want to remove the whole RTS code, I prefer doing it in a separate PR. Does it acceptable to you?

Reply:
@wzamazon meant to not change anything, just leave as what it was. Updated.